server/plugins/router/router.module.js

```diff
@@ -11,6 +11,7 @@ const internals = {
         renderer: '/{url*}',
         staticAssets: '/assets/{path*}',
         internalApi: '/internalapi/{path*}',
+        storeFrontApi: '/api/storefront/{path*}',
```

Comment:
The term we use in docs is “Storefront API”, so perhaps this should be `storefrontAPI` 🍹

Reply:
Will rename. thanks